def RaiseError(exception: Exception):
    raise exception
